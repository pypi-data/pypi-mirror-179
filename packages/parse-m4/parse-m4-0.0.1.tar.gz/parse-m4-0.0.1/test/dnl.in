foo dnlfoo
bar dnl bar
abc dnl
xyz
xxxdnl
