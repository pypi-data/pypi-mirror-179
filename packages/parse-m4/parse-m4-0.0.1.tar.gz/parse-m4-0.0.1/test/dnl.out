foo dnlfoo
bar abc xyz
xxxdnl
